### PR TITLE
Linking in prerequisite sections

### PR DESCRIPTION
**Pull request by rudri182**
I know there are so many commits and there can be some duplication in commits. But I was very confused doing cherry picking. So let me know whether I have done correctly or not. Cherry-pick from PR #793

### CODE REVIEW

**Comment by rudri182** [comment]
If this branch is working properly then may I add the files to this branch?

**Comment by saberprashant** [comment]
Hi @rudri182, well, in this PR, you've cherry-picked the changes of only 2 files. But in your PR #793, you changed a total of 15 files. So, my question is, are these your final changes? or you want to do some more changes?

**Comment by rudri182** [comment]
> Hi @rudri182, well, in this PR, you've cherry-picked the changes of only 2 files. But in your PR #793, you changed a total of 15 files. So, my question is, are these your final changes? or you want to do some more changes?

@saberprashant Thanks for noticing it.
Yes, I have more files to add. But I am trying it for the first time. So I was not sure about it. Now I will upload more. 😊

**Comment by rudri182** [comment]
I have tried my best and modified outline as the comments by @praveenscience in the branch **Converting check for understanding** PR #793. And added the questions in the Stack.md which is the main task for issue #700. Please review it.

**Comment by rudri182** [comment]
> Are you changing prerequisites or working on "check for understanding"s? Those are 2 separate issues.

I am working on issue #700 but in PR #793 @praveenscience suggested some changes in prerequisites section that was pushed by mistake. After that, I also tried to remove those files by `git -rm <filename>` and then pushed them but still, they are appearing. I have worked on prerequisites sections previously so I am working on **CFU** section here.

**Comment by alodahl** [comment]
Keep trying!  This is a real life problem, and these are great skills to have for an engineering career. Its also a good reminder to check which branch you're on before you commit or push!

**Comment by rudri182** [comment]
I have resolved the conflicts. I will make a new branch for the  **prerequisites** section and upload these there.

**Comment by rudri182** [comment]
May I remove the stack.md file from here and keep other things as it is? So there is no need to create another branch. As I have already created a new branch for **Check For Understanding** section.

File: react-js/react.md [hunk 1]
```diff
@@ -4,9 +4,8 @@
 8 Hours
 
 ### Prerequisites
-
-* [Javascript I - VI](https://github.com/Techtonica/curriculum/tree/master/javascript)
-* [jQuery Topic Outline](https://github.com/Techtonica/curriculum/tree/master/jquery)
+* [Javascript I - VI](../javascript)
+* [jQuery Topic Outline](../jquery)
```
Comment (alodahl) [inline react-js/react.md]: Please link to the jquery file

Comment (rudri182) [reply react-js/react.md]: Linking done 👍

File: deploying/deploying.md [hunk 2]
```diff
@@ -5,20 +5,17 @@
...
-- Download and install the Heroku CLI with `brew install heroku/brew/heroku`
-- Node.js and npm must be installed
-- An existing Node.js app
-- [JS I - VI](/javascript)
-- [Node](/node-js/node-js.md)
-- [Express](/express-js/express.md)
-- [MongoDB](/databases/mongo-db.md)
+Node.js and npm must be installed
+An existing Node.js app
+- [JS I - VI](https://github.com/Techtonica/curriculum/tree/master/javascript)
```
Comment (Kundan28) [inline deploying/deploying.md]: `JS I - VI` link could be made short too...

File: node-js/node-js.md [hunk 3]
```diff
@@ -5,11 +5,11 @@
...
-- [JavaScript I - VI lesson](/javascript)
-- [How the Internet Works](/requests-and-responses/requests-and-responses.md)
-- [AJAX](/ajax/ajax.md)
-- [Git and Version Control](/git-version-control/git-version-control.md)
-- [jQuery](/jquery/jquery.md)
+- [JavaScript I - VI lessons](../javascript)
+- [How the Internet Works](../requests-and-responses/requests-and-responses.md)
+- [AJAX](../ajax/ajax.md)
+- [Git and Version Control](../git-version-control/git-version-control.md)
+- [jQuery](https://github.com/Techtonica/curriculum/tree/master/jquery)
```
Comment (Kundan28) [inline node-js/node-js.md]: This link couldn't be made short as the `JavaScript I - VI` ?

File: open-source/open-source.md [hunk 4]
```diff
@@ -4,9 +4,9 @@
 1 hour
 
 ### Prerequisites
+- ["Git and version control" lesson](../git-version-control/git-version-control.md)
```
Comment (Kundan28) [inline open-source/open-source.md]: You can remove the blank line number 8.

**Comment by rudri182** [comment]
I have made the changes as suggested by @Kundan28. Please suggest if there are more changes to be made. 😊

**Comment by rudri182** [comment]
👍😊

**Comment by alodahl** [comment]
@rudri182 , does this fix all the prereq sections in https://github.com/Techtonica/curriculum/issues/731? if so please rename and describe your issue so we can give you points without confusion.

**Comment by rudri182** [comment]
> @rudri182, does this fix all the prereq sections in #731? if so please rename and describe your
> pr so we can give you points without confusion.

Yes, they are. Still, if there is missing something then I'm happy to solve it. 😊

**Comment by alodahl** [comment]
Please fix your title and description.

**Pull request merged**